core/api/graphql/mutation_create_artifact.graphql

```diff
@@ -33,14 +47,47 @@ mutation CreateArtifact(
         sequenceClientID: $sequenceClientID,
         }
     ) {
-        artifact {
-            id
-            state
-            artifactSequence {
-                latestArtifact {
-                    id
-                }
-            }
-        }
+        ...CreatedArtifact
+    }
+}
+
+# CreateArtifactWithoutTags implements the CreateArtifact mutation
+# for older server versions that do not support artifact tags.
+mutation CreateArtifactWithoutTags(
+    $entityName: String!,
+    $projectName: String!,
+    $artifactTypeName: String!,
+    $artifactCollectionName: String!,
+    $runName: String,
+    $digest: String!,
+    $description: String,
+    $aliases: [ArtifactAliasInput!],
+    $metadata: JSONString,
+    $ttlDurationSeconds: Int64,
+    $historyStep: Int64,
+    $distributedID: String,
+    $clientID: ID!,
+    $sequenceClientID: ID!,
+) {
+    createArtifact(input: {
+        entityName: $entityName,
+        projectName: $projectName,
+        artifactTypeName: $artifactTypeName,
+        artifactCollectionName: $artifactCollectionName,
+        runName: $runName,
+        digest: $digest,
+        digestAlgorithm: MANIFEST_MD5,
+        description: $description,
+        aliases: $aliases,
+        metadata: $metadata,
+        ttlDurationSeconds: $ttlDurationSeconds,
+        historyStep: $historyStep,
+        enableDigestDeduplication: true,
+        distributedID: $distributedID,
+        clientID: $clientID,
+        sequenceClientID: $sequenceClientID,
+    }
+    ) {
+        ...CreatedArtifact
```

Comment:
context: This is nearly identical to the `CreateArtifact` snippet above, just with 2 lines omitted:
- the `$tags: [TagInput!]` arg
- the `tags: $tags` input field